presto-hive/src/main/java/com/facebook/presto/hive/HivePartitionManager.java

```diff
@@ -282,6 +277,24 @@ else if (type instanceof TinyintType
                 .orElseThrow(() -> new TableNotFoundException(tableName));
     }
 
+    public static HivePartition parsePartition(
+            SchemaTableName tableName,
+            String partitionId,
```

Comment:
s/partitionId/partitionName for consistency with `partitionName` parameter in `extractPartitionKeyValues` method